script/header/pageReload.js

```diff
@@ -0,0 +1,14 @@
+import { getQuerySelector } from "../../utils/js/getElements.js"
+
+function pageReload() {
+  const newsLogo = getQuerySelector(document, '#header-news-icon');
+  newsLogo.addEventListener('click', () => {
+    location.reload(true);
+    console.log("hello");
+  });
```

Comment:
console 로그는 커밋할 시에 제거해주면 좋을거 같아요!! 
아마 reload 함수에 파라마터를 안줘도 default로 true로 작동하는것으로 알고 있어서

```suggestion
  newsLogo.addEventListener('click', () => location.reload);
```
로 수정해보면 어떨까요??

Reply:
아직 디버깅이 낯설어서 console 로그로 찍어봤는데 ,, 까먹고 지우지 않았네요 ㅜ ㅜ .. 놓친 부분을 잡아주셔서 감사합니다 !!
이렇게 쓰니 코드가 훨씬 더 깔끔한 것 같네요 !!